bluejeans/src/bluejeans_create.js

```diff
@@ -24,6 +24,16 @@ async function _command(params = {}, commandText, secrets = {}) {
     };
   }
 
+  const startTime = new Date(start).getTime();
+  const endTime = new Date(end).getTime();
+
+  if (endTime <= startTime) {
```

Comment:
Can you also check if the start time is not in the past?

Reply:
:+1: